lib/csv.rb

```diff
@@ -1131,8 +1131,8 @@ def open(filename, mode="r", **options)
 
     #
     # :call-seq:
-    #   parse(string) -> array_of_arrays
-    #   parse(io) -> array_of_arrays
+    #   parse(string) -> array_of_arrays or csv_table
+    #   parse(io) -> array_of_arrays or csv_table
```

Comment:
How about adding the following new entries instead of changing the current entries?

```ruby
parse(string, headers: ..., **options) -> csv_table
parse(io, header: ..., **options) -> csv_table
```

Reply:
Can you do it?

Reply:
Done.